mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/config/StaticPropertiesConfigurationFactory.java

```diff
@@ -27,6 +29,8 @@ public class StaticPropertiesConfigurationFactory implements ConfigurationFactor
     public StaticPropertiesConfigurationFactory(Properties props) {
         delegate = new ConfigurationObjectFactory(props);
         delegate.addCoercible(new MetricsCoercer(props));
+        delegate.addCoercible(new LeaderElectorFactoryCoercer());
```

Comment:
tried to avoid these, but our skife config-magic version requires I create these.